core/data-resolver/UserService.ts

```diff
@@ -84,16 +84,13 @@ const changePassword = async (passwordData: DataResolver.PasswordData): Promise<
     }
   })
 
-const invalidateToken = async (refreshToken: string): Promise<Task> =>
-  TaskQueue.execute({
-    url: processLocalizedURLAddress(config.users.refresh_endpoint),
-    payload: {
-      method: 'POST',
-      mode: 'cors',
-      headers,
-      body: JSON.stringify({ refreshToken })
-    }
-  })
+const invalidateToken = async (refreshToken: string): Promise<any> =>
```

Comment:
it should not be any
it returns object like that:
```json
{"code":200,"result":"mwgdkaspawubhrmyc64oq4qj72ng9yyr","meta":{"refreshToken":"6474ccd3931747fd7bbf08d27a4f15bcc8dec730acee04882b6a046d7a768a165e55e2949d739adb6b103c8b594c543a977342b13768a2f74d2a8baa896d3a0d795862c02f7d35c3bfaa852ae8718330ce9c0c71af859c11cf354a408a19b7fd37ce0a434b61dc7e20c4dbf895a72721a7b883ed126cd3826c92adef00313d2a5680d5b19b5e4ae28f5b21f2f75484c761fed56c741af1f3976efb3f6330462d96c472c4812ac494"}}
```
so we should return just string with resp.result. Also this endpoint is not invalidatingToken - it refreshes it, so creating new current-token for user session, so it would be nice to change this method this way :)

Reply:
Yeah, sure, but there is also `refreshToken` in that response which is used in `USER_TOKEN_CHANGED` mutation. So I can just create a type/interface for that.

Reply:
Actually no, refresh token is not changing, passed only by meta data which we can ignore here. We're giving it as a post params to get ner current-token. I really think that just returning string with new current-token and renaming method is a proper way to go here :)

Reply:
@patzick I've just tested it - doesn't work, I can't even login without that. Basically this is the only place where `refreshToken` is set

Reply:
@patzick Updated. You are right, I've noticed that `refreshToken` during the login & it works fine. Thanks!